clients/ts/tsconfig.base.json

```diff
@@ -14,6 +14,15 @@
         "suppressImplicitAnyIndexErrors": true,
         "noEmitOnError": true,
         "stripInternal": true,
-        "lib": [ "es5", "es2015.promise", "es2015.iterable", "dom" ]
+        "lib": [ "es5", "es2015.promise", "es2015.iterable", "dom" ],
+        "baseUrl": ".",
+        "paths": {
```

Comment:
This section provides mappings so TypeScript can find the packages. This is really just used by `signalr-protocol-msgpack` to find `signalr`. This has **no effect** on the generated code, it's just so TypeScript can find the definition files for the package.